Source/Common/gdcmByteSwap.txx

```diff
@@ -106,7 +106,7 @@ void Swap4(T &a, SwapCode const &swapcode)
 {
 #ifndef GDCM_WORDS_BIGENDIAN
   if ( swapcode == gdcm::SwapCode::BigEndian || swapcode == gdcm::SwapCode::BadBigEndian )
-    a = (T)(( a << 8 ) | ( a >> 8 ));
+    a = ( a << 8 ) | ( a >> 8 );
```

Comment:
could you include the warning/error reported here ? Here is what I see using git blame: 1033e91199

Reply:
There is no warning/error. This patch comes from me comparing the code inside `GDCM_WORDS_BIGENDIAN` versus the code outside it.  ex, in master, why are these two lines different?

https://github.com/malaterre/GDCM/blob/008e437d6ea81979bfa23301ae6e62f82272fe64/Source/Common/gdcmByteSwap.txx#L109

https://github.com/malaterre/GDCM/blob/008e437d6ea81979bfa23301ae6e62f82272fe64/Source/Common/gdcmByteSwap.txx#L112

Reply:
Do you see what I mean now?